spec/requests/requests_requests_spec.rb

```diff
@@ -48,6 +48,43 @@
           expect(response).to have_http_status(:not_found)
         end
       end
+
+      context 'When organization has a default storage location' do
+        let(:request) { create(:request, organization: create(:organization, default_storage_location: 1)) }
+        it 'shows the column Default storage location inventory' do
+          get request_path(request)
+
+          expect(response.body).to include('Default storage location inventory')
+        end
+      end
+
+      context 'When organization does not have a default storage location' do
+        let(:request) { create(:request, organization: organization) }
+        it 'shows the column Default storage location inventory' do
```

Comment:
The text on this it clause should match what you are testing ... that it does *not* show the default storage location inventory.